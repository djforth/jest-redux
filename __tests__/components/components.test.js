import React from 'react';

import Component from '../__dummy__/components';
import { shallow } from 'enzyme';

import ComponentBasics from 'components';

describe('<Component />', () => {
  let component;
  beforeAll(() => {
    component = shallow(<Component />);
  });

  ComponentBasics(() => component);
});
