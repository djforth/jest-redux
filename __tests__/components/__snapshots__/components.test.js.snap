// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component /> Basic component tests snapshot matches 1`] = `
<div>
  foo
</div>
`;
